src/components/Navbar/SearchDrawer/SearchDrawer.module.scss

```diff
@@ -97,3 +97,15 @@
   margin-top: constants.$navbar-height;
   padding: var(--spacing-medium);
 }
+
+.spinnerContainer div {
+  margin: 0 auto;
+}
+
+.internalContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: calc(92% - #{constants.$navbar-height});
```

Comment:
so percentage in height doesn't work exactly like percentage for width. tl;dr you'd want to use `vh`